### DISCUSSION TITLE
Jekyll_app1htmlFile/test/2002/09/18/html_notranslate_test

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Jekyll_app1htmlFile#17] [Komentarze]
# Jekyll_app1htmlFile/test/2002/09/18/html_notranslate_test

Test…

http://localhost:4000/Jekyll_app1htmlFile/test/2002/09/18/html_notranslate_test.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
